mrscake/src/components/orderDetails/CartItem.js

```diff
@@ -0,0 +1,88 @@
+import React, {useState} from 'react';
+import {connect} from 'react-redux';
+//import {shortenTitle} from "../../pipes/shortenTitle";
+//import {formatMoney} from "../../pipes/priceFormatter";
+import './CartItem.scss';
+import { decrementCartQuantity, incrementCartQuantity, removeProductToCart} from "../../actions";
+
```

Comment:
which component is it? Class or functional component?

Reply:
its constant, has the same fucntion as a function

Reply:
it has the same function as function component